cmd/wrap_test.go

```diff
@@ -1,12 +1,80 @@
 package cmd
 
 import (
+	"bytes"
+	"io/ioutil"
 	"testing"
 
 	. "github.com/smartystreets/goconvey/convey"
+	"github.com/spf13/viper"
 )
 
 func TestWrap(t *testing.T) {
-	// Any tests we can get?
-	Convey("wrap", t, nil)
+	Convey("wrap", t, func() {
+		viper.Reset()
+		viper.Set("kubeconfigfiles.directory", "/tmp/")
+
+		Convey("returns error when no -e flag is set", func() {
+			viper.Set("environments", map[string]interface{}{
+				"edge-cmh": "kubeconfig-truss-nonprod-cmh",
+			})
+
+			cmd := rootCmd
+			viper.BindPFlag("TRUSS_ENV", cmd.PersistentFlags().Lookup("env"))
```

Comment:
@jgramoll I noticed I had to set this everywhere so that `getKubeconfigName` did the right thing. Otherwise, it bails early with `""`.

What's the intent behind `TRUSS_ENV`? And why does `getKubeconfigName` short-circuit when it's not provided?